test/travis/setup_ldap.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+
+# Script taken from https://github.com/piwik/plugin-LoginLdap/blob/master/tests/travis/setup_ldap.sh
+# install LDAP
+echo "Installing LDAP..."
+sudo apt-get update > /dev/null
+if ! sudo apt-get install slapd ldap-utils -y -qq > /dev/null; then
+    echo "Failed to install OpenLDAP!"
+fi
+
+# configure LDAP
+echo ""
+echo "Configuring LDAP..."
+
+mkdir -p $TRAVIS_BUILD_DIR/ldap
+sudo chmod -R 777 $TRAVIS_BUILD_DIR/ldap
+
+ADMIN_USER=deathofrats
+ADMIN_PASS=squeak
+ADMIN_PASS_HASH=`slappasswd -h {md5} -s $ADMIN_PASS`
+BASE_DN="dc=ankh-morpork,dc=dw"
+
+STR_OID="1.3.6.1.4.1.1466.115.121.1.15"
+VIEW_OID="2.16.840.1.113730.3.1.1.1"
+ADMIN_OID="2.16.840.1.113730.3.1.1.2"
+SUPERUSER_OID="2.16.840.1.113730.3.1.1.3"
+
+sudo ldapmodify -Y EXTERNAL -H ldapi:/// <<EOF
+dn: cn=config
+changetype: modify
+replace: olcLogLevel
+olcLogLevel: -1
+#-
+#add: olcDisallows
+#olcDisallows: bind_anon
+EOF
+
+sudo ldapadd -Y EXTERNAL -H ldapi:/// <<EOF
+# database
```

Comment:
I won't be able to use Variable-substitution in those files. Therefore I'd have to change usernames and passwords in there manually which would remove the possibility to adapt to different setups. 

I'd rather avoid that!